### PR TITLE
ContentUpdate and Removal endpoints integration

### PR DESCRIPTION
**Pull request by lukasz-falda**

### CODE REVIEW

File: src/LuigisBoxBundle/Util/AuthenticationUtil.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        string $endpoint,
+        \DateTimeInterface $date
+    ): string {
+        $contentType = self::CONTENT_TYPE;
+
+        $dateString = self::formatDate($date);
+        $data = "{$httpMethod}\n{$contentType}\n{$dateString}\n{$endpoint}";
+
+        return trim(base64_encode(hash_hmac('sha256', $data, $privateKey, true)));
+    }
```
Comment (lukasz-falda) [inline src/LuigisBoxBundle/Util/AuthenticationUtil.php:40]: digest is method copied from documentation

File: composer.json [hunk 2]
```diff
@@ -8,11 +8,13 @@
...
     "symfony/validator": "^4.1",
-    "symfony/http-kernel": "^4.1"
+    "symfony/http-kernel": "^4.1",
+    "php-http/guzzle6-adapter": "^1.1",
+    "symfony/serializer": "^4.4"
   },
   "require-dev": {
     "roave/security-advisories": "dev-master",
-    "phpunit/phpunit": "^9.0",
+    "phpunit/phpunit": "^8.0",
```
Comment (lukasz-falda) [inline composer.json:17]: Some issue on PhpStorm while using ^9.0 version.

File: phpunit.xml.dist [hunk 3]
```diff
@@ -19,12 +19,7 @@
...
     </testsuites>
 
-    <filter>
-        <whitelist processUncoveredFilesFromWhitelist="true">
-            <directory>./src/LuigisBoxBundle</directory>
-            <exclude>
-                <directory>./src/LuigisBoxBundle/Tests</directory>
-            </exclude>
-        </whitelist>
-    </filter>
```
Comment (lukasz-falda) [inline phpunit.xml.dist:29]: Not need it.

File: src/LuigisBoxBundle/Tests/Unit/Factory/ContentUpdateFactoryTest.php [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+     * @test
+     */
+    public function prepareRequestSuccessfully(): void
+    {
+        $objects = $this->getObjects();
+        $factory = $this->getFactory($objects);
+
+        $request = $factory->prepareRequest($objects);
+
+        $this->assertSame('host//v1/content', $request->getUri()->getPath());
```
Comment (jrawska) [inline src/LuigisBoxBundle/Tests/Unit/Factory/ContentUpdateFactoryTest.php:27]: double // ?

**Review by jrawska — APPROVED** [review]

File: src/LuigisBoxBundle/Tests/Unit/Factory/ContentUpdateFactoryTest.php (hunk 4 above)
Comment (lukasz-falda) [reply src/LuigisBoxBundle/Tests/Unit/Factory/ContentUpdateFactoryTest.php:27]: Its by mistake. I'll fix it

Comment (lukasz-falda) [reply src/LuigisBoxBundle/Tests/Unit/Factory/ContentUpdateFactoryTest.php:27]: Fix at https://github.com/answear/luigis-box-bundle/pull/4/files#diff-3f1939ab2f8840dbb31bd46b78fabd0c

**Review by bmulawa — APPROVED** [review]

**Pull request merged**